taskcluster/config.yml

```diff
@@ -64,32 +64,32 @@ workers:
             provisioner: '{trust-domain}-{level}'
             implementation: docker-worker
             os: linux
-            worker-type: '{alias}'
+            worker-type: 'b-linux-large-gcp-d2g'
         b-linux-large-gcp-300gb:
             provisioner: '{trust-domain}-{level}'
             implementation: docker-worker
             os: linux
-            worker-type: '{alias}'
+            worker-type: 'b-linux-large-gcp-d2g-300gb'
         b-linux-large-gcp-1tb-32-256:
             provisioner: '{trust-domain}-{level}'
             implementation: docker-worker
             os: linux
-            worker-type: b-linux-large-gcp-1tb-32-256-d2g
+            worker-type: 'b-linux-large-gcp-d2g-300gb'
```

Comment:
@bhearsum I don't understand this change. Why everything is 300gb now? I guess it will break the pipeline steps that need 1TB of disk and other specific machine requirements. Or the `worker-type` setting doesn't control those?

Reply:
Sorry...this part was an oversight! My testing was all done with 300gb workers. I'll fix this up.